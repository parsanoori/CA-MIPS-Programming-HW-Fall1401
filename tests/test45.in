70
617213688
767579752
1018446130
290530948
386129053
304727617
586458393
381556851
238813611
893923166
1029580625
201260466
953017138
112937831
461565531
250392798
896408674
581431854
562352380
609079258
891791615
171097079
1038406945
393953401
603167530
70269494
818319105
778169635
18819252
482197446
557215472
727152478
595171503
13915890
267361074
962647671
767212074
10896155
463334724
773201030
772360184
544727057
427188665
768420075
995325335
510747111
668403799
795657732
923078198
845473156
318516155
1025166114
646329427
569350641
729793159
712147546
166432270
914205408
375353102
247630221
936712359
601472429
444086258
365775619
858474191
1065505120
546745587
26371081
481245339
82302004