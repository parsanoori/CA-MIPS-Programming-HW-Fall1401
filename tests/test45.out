10896155
13915890
18819252
26371081
70269494
82302004
112937831
166432270
171097079
201260466
238813611
247630221
250392798
267361074
290530948
304727617
318516155
365775619
375353102
381556851
386129053
393953401
427188665
444086258
461565531
463334724
481245339
482197446
510747111
544727057
546745587
557215472
562352380
569350641
581431854
586458393
595171503
601472429
603167530
609079258
617213688
646329427
668403799
712147546
727152478
729793159
767212074
767579752
768420075
772360184
773201030
778169635
795657732
818319105
845473156
858474191
891791615
893923166
896408674
914205408
923078198
936712359
953017138
962647671
995325335
1018446130
1025166114
1029580625
1038406945
1065505120